src/test/java/teammates/test/driver/GaeSimulation.java

```diff
@@ -183,24 +182,6 @@ public void loginAsStudentInstructor(String userId) {
         assertFalse(user.isAdmin);
     }
 
-    /**
-     * Returns an {@link teammates.ui.controller.Action} object that matches the parameters given.
-     *
-     * @param parameters Parameters that appear in a HttpServletRequest received by the app.
-     */
-    public teammates.ui.controller.Action getLegacyActionObject(String uri, String... parameters) {
-        InvocationContext ic = invokeWebRequest(uri, parameters);
```

Comment:
You can remove this method and the field `sc` of this class as well. Also you can remove the `httpunit` test dependency in `build.gradle`.